CODE_OF_CONDUCT.md

```diff
@@ -40,6 +40,14 @@ Project maintainers are responsible for clarifying the standards of
 acceptable behavior and are expected to take appropriate and fair
 corrective action in response to any instances of unacceptable behavior.
 
+Members of the official team are required to disclose any interests 
+that may conflict against the interests of the privacytools.io website and/or community. 
+If a team member has a private interest (for example, a job at a company which is listed 
+or wants to be listed on the privacytools website), then it is seen as their duty to disclose this interest. 
```

Comment:
I think this requires more definition on what is a conflict of interest. For example [I gave a long list at the forum](https://forum.privacytools.io/t/preventing-privacytools-conflicts-of-interest-ensuring-privacytools-integrity/2517/25?u=mikaela) and while I still was in the Matrix team chat, I understood that there the list was disapproved and from what I have talked with people, they don't consider anything else than some associations a CoI (which by the way has been discussed previously in very long comments in https://github.com/privacytoolsIO/privacytools.io/issues/1062#issuecomment-516395200).

To clarify the question, where is the line between friendliness with a project and a CoI? When does a relationship with a person in some sort of connection with a project become a CoI? How about being enthustiastic about a project?

Is the definition just money in the equation?

Reply:
changed it so it implies that whether something is a COI shall be determined on a case by case basis.